doc/modules/ROOT/pages/nrepl-api/ops.adoc

```diff
@@ -971,6 +971,7 @@ Optional parameters::
 
 Returns::
 * `:elapsed-time` a report of the elapsed time spent running all the given namespaces. The structure is ``:elapsed-time {:ms <integer> :humanized <string>}``.
+* `:fail-fast` If true, the tests will be considered complete after the first test has failed or errored.
```

Comment:
Just keep in mind there's no boolean type in bencode, so `true` is probably not the best way to describe this. But I get the idea. :-)

Reply:
Maybe a small change like `When true or the string "true"`...